lib/mayaUsd/fileio/utils/xformStack.cpp

```diff
@@ -529,6 +528,67 @@ UsdMayaXformStack::OpClassList UsdMayaXformStack::FirstMatchingSubstack(
 
 const UsdMayaXformStack& UsdMayaXformStack::MayaStack()
 {
+#if USD_SUPPORTS_SPLINES
+    static UsdMayaXformStack mayaStack(
+        // ops
+        {
+            UsdMayaXformOpClassification(
+                UsdMayaXformStackTokens->translate, UsdGeomXformOp::TypeTranslate),
+            UsdMayaXformOpClassification(
+                UsdMayaXformStackTokens->translateX, UsdGeomXformOp::TypeTranslateX),
+            UsdMayaXformOpClassification(
+                UsdMayaXformStackTokens->translateY, UsdGeomXformOp::TypeTranslateY),
+            UsdMayaXformOpClassification(
+                UsdMayaXformStackTokens->translateZ, UsdGeomXformOp::TypeTranslateZ),
+            UsdMayaXformOpClassification(
+                UsdMayaXformStackTokens->pivot, UsdGeomXformOp::TypeTranslate),
+            UsdMayaXformOpClassification(
+                UsdMayaXformStackTokens->rotatePivotTranslate, UsdGeomXformOp::TypeTranslate),
+            UsdMayaXformOpClassification(
+                UsdMayaXformStackTokens->rotatePivot, UsdGeomXformOp::TypeTranslate),
+            UsdMayaXformOpClassification(
+                UsdMayaXformStackTokens->rotate, UsdGeomXformOp::TypeRotateXYZ),
```

Comment:
Just want to  point out that this will match XYZ rotation order but not any other order. That is "fine" as that is Maya default order, but it has always meant this does not match any other order.

Reply:
As you said, this is how it has always been, I was just pointing it out.